config/tst.config.yml

```diff
@@ -21,12 +21,12 @@ driver: chrome
 # This option only applies when the driver is set to 'chrome' or 'firefox'.
 # Phantomjs is a headless only browser, and option is meaningless for
 # browserstack
-headless: false
+headless: true
 
 # Add a pause (in seconds) between steps so you can visually track how the
 # browser is responding. Only useful if using a non-headless browser. The
 # default is 0
-pause: 0
+pause: 5
```

Comment:
You might have a contradiction here @andrewhick . `headless` is true which means you wouldn't see it, but `pause` is normally used because you want to watch at a reasonable speed what is happening.

Reply:
Good point, I've been switching between headless and non-headless so that's probably why. I'll switch it back to 0 for now.